test/jdk/jdk/internal/vm/Continuation/Fuzz.java

```diff
@@ -91,6 +91,9 @@ public static void main(String[] args) {
             throw new SkippedException("Test is unstable with slowdebug bits "
                                        + "on macosx-aarch64");
         }
+        if (Platform.isPPC()) {
+            COMPILATION_TIMEOUT = COMPILATION_TIMEOUT * 2;
```

Comment:
I guess this won't compile. COMPILATION_TIMEOUT is final.

Reply:
Yes correct, it cannot be final any more.

Reply:
Well it could be final if you would add `(Platform.isPPC() ? 2 : 1) * ...` to the initialization. But I'm indifferent.